locations/spiders/we_nl.py

```diff
@@ -0,0 +1,37 @@
+import re
+
+import scrapy
+
+from locations.dict_parser import DictParser
+from locations.hours import DAYS_NL, OpeningHours, sanitise_day
+
+
+class WeNLSpider(scrapy.Spider):
+    name = "we_nl"
+    item_attributes = {"brand": "We", "brand_wikidata": "Q1987861"}
+
+    def start_requests(self):
+        yield scrapy.Request(
```

Comment:
This can just be `start_urls = []`. For example [here](https://github.com/alltheplaces/alltheplaces/blob/1bf9caadafbe3d27f47006beaa6164dd9b90cba6/locations/spiders/hm.py#L10).